app/advancedfilters/advancedFilters.html

```diff
@@ -107,6 +89,39 @@
       </div>
     </li>
 
+    <!-- GENE PRODUCT TYPE -->
+    <li uib-dropdown auto-close="outsideClick" is-open="status.isopenGPT">
+      <button type="button" class="btn btn-default quickgo-btn  icon" ng-class="{'icon-functional active-filter' : isActiveFilter('gpType')}" data-icon="/" uib-dropdown-toggle>
+        Gene Product Type <span class="caret"></span>
+        <br>
+      </button>
+      <div class="advanced-filter-content" uib-dropdown-menu>
+        <div class="form-group">
+          <form>
+            <ul>
+              <li class="checkbox">
+                <label>
+                  <input type="checkbox" ng-model="filters.gpType['protein']"> proteins</label>
+              </li>
+              <li class="checkbox">
+                <label>
+                  <input type="checkbox" ng-model="filters.gpType['rna']"> RNAs</label>
+              </li>
+              <li class="checkbox">
+                <label>
+                  <input type="checkbox" ng-model="filters.gpType['complex']"> complexes</label>
+              </li>
+            </ul>
+            <hr>
+            <div class="pull-right">
+              <button class="btn quickgo-btn" type="button" ng-click="resetGPType()">Reset</button>
+              <button class="btn quickgo-btn" type="button" ng-click="updateFilters()">Apply</button>
+            </div>
+          </form>
+        </div>
+      </div>
+    </li>
+
```

Comment:
What is the expected behaviour of this filter? I tried one by one the three possibilites, proteins, RNAs and complexes, and nothing happened, the total number of annotations remained the same. And the tick does not stay. I checked the maser branch and that one has only proteins so, I if I select RNAs, I get 0, same should happen in this branch.